openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -61,24 +65,30 @@ import {
   hasEditAccess,
 } from '../../utils/CommonUtils';
 import { filterEntityAssets } from '../../utils/EntityUtils';
-import { hasPemission } from '../../utils/PermissionsUtils';
+import { checkPermission, hasPemission } from '../../utils/PermissionsUtils';
```

Comment:
Can we use only the checkPermission method and remove the hasPemission method?

Reply:
made suggested changes